test/integration/Cardano/WalletSpec.hs

```diff
@@ -64,4 +66,4 @@ spec = do
         threadDelay 1000000
         (handle,) <$> (mkWalletLayer
             <$> MVar.newDBLayer
-            <*> HttpBridge.newNetworkLayer "testnet" port)
+            <*> HttpBridge.newNetworkLayer Testnet port)
```

Comment:
Seeing this… 🤔 won't this lead to inconsistencies between the network layers and the CLI param in future tests?

Reply:
Hmmm.. I am not sure to see what you mean here? There's no more "--network" in the CLI, and the CLI itself reads the environment to get this network value.

Reply:
If we call `mkStdTx` the protocol magic it uses will depend on `network` and the ENV var. If we ever create a network layer with a custom network, like here (`Testnet`), we must never use the two together.

Would it be possible to have *one* `network` for the entire integration tests?

Also:
- Does the choice of network matter for the integration tests?

Reply:
@Anviking  Hmm... That's not possible no, because we are running some integration tests against testnet, but some against a local cluster (the scenarios) because we need the ability to have faucets wallets there, and submit transactions at will.

So yeah, if the application is configured with a protocol magic / network that is different from the one used by the bridge, we can an error; but that's pretty much what we have already anyway.